Automattic/wp-calypso
# Gutenboarding: add header

Adds a basic header canvas with an example button.

As next steps outside this PR we need to somehow ensure that `.layout__content` and Calypso CSS reset styles don't collide with block editor.

In this PR we introduce a bunch of [Gutenberg stylesheets](https://github.com/WordPress/gutenberg/tree/26795cac83cbcd4a78a2045c444d9d30ff2f0590/assets/stylesheets) in Calypso, but those should be in future imported via `base-styles` package, being worked at https://github.com/WordPress/gutenberg/pull/17883.

## Testing

- Visit `/gutenboarding` in development env and confirm that you see a header that doesn't do anything yet. :-) 
- Confirm that CSS reset isn't leaking to rest of Calypso
